telemetry/packages/ui/app/components/views/state-machine/index.tsx

```diff
@@ -1,19 +1,38 @@
 import { StateMachineFlowChart } from './flow-chart';
-import { PodState } from '../../shared/pod-state';
+import { PodState } from '@/components/shared/pod-state';
 import { useCurrentPod } from '@/context/pods';
 
+import { StateButton } from './dev-components/debug-btn';
+import React, { useState } from 'react';
+import { PodStateType, ModeType } from '@hyped/telemetry-constants';
+
 /**
  * The state machine view which displays the current state of the pod and the flow chart describing the state machine.
  */
 export const StateMachine = () => {
   const {
-    pod: { id, podState },
+    pod: { id },
   } = useCurrentPod();
 
+  // Debug state change buttom
+  const [state, setState] = useState<PodStateType>('IDLE');
+  const handleStateChange = (newState: PodStateType) => {
+    setState(newState);
+  };
+
+  const [mode, setMode] = useState<ModeType>('ALL_SYSTEMS_ON');
+  const handleModeChange = (newMode: ModeType) => {
+    setMode(newMode);
+  };
+
   return (
-    <div className="h-full pb-64">
+    <div className="h-full">
       <PodState podId={id} />
```

Comment:
I don't think we'll keep this, we'll probably just have the entire view taken up with the diagram instead